### ISSUE TITLE
version not printed

### ISSUE THREAD

**Issue by leodotcloud** [aquasecurity/kube-bench#545]
```
# export kube_bench_tag=0.2.2
# wget https://github.com/aquasecurity/kube-bench/releases/download/v${kube_bench_tag}/kube-bench_${kube_bench_tag}_linux_amd64.tar.gz
# tar xvfz kube-bench_0.2.2_linux_amd64.tar.gz
LICENSE
README.md
kube-bench
# ./kube-bench version

#
```

**Comment by lizrice** [comment]
Yes, I can reproduce this. However, the version is there if you run from the Docker image: 
```
$ docker run -it --rm aquasec/kube-bench:latest version
v0.2.2
```
which suggests that the code for the `version` command itself is working, but somehow the version number is not being populated into the `cmd.KubeBenchVersion` variable by GoReleaser.

**Comment by robertojrojas** [comment]
This should be fixed in PR #540

**Comment by lizrice** [comment]
We think this is fixed - we'll know for sure when we cut the release! Let's close and reopen if it's still a problem after we cut 0.3.0
